.env.example

```
@@ -1,3 +1,25 @@
 HUME_SECRET_KEY=
-NEXT_PUBLIC_HUME_CONFIG_ID=
\ No newline at end of file
+NEXT_PUBLIC_HUME_API_KEY=
+HUME_SERVER_API_KEY=
+NEXT_PUBLIC_HUME_CONFIG_ID=
+
+# Email (Trigger Email extension destination)
+LEADS_EMAIL=jsb@synsol.ch
+
+# Consent gating
+CONSENT_MODE=optional   # required|optional|off
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)